Specs/Core/TrustedServersSpec.js

```diff
@@ -0,0 +1,73 @@
+/*global defineSuite*/
+defineSuite([
+    'Core/TrustedServers'
+], function(
+    TrustedServers) {
+    'use strict';
+
+    afterEach(function() {
+        TrustedServers.clear();
+    });
+
+    it('add without argument throws', function() {
+        expect(function() {
+            TrustedServers.add();
+        }).toThrowDeveloperError();
+    });
+
+    it('remove without argument throws', function() {
+        expect(function() {
+            TrustedServers.remove();
+        }).toThrowDeveloperError();
+    });
+
+    it('isTrusted without argument throws', function() {
+        expect(function() {
+            TrustedServers.contains();
+        }).toThrowDeveloperError();
+    });
+
+    it('without a port', function() {
+        TrustedServers.add('cesiumjs.org');
+        expect(TrustedServers.contains('http://cesiumjs.org/index.html')).toBe(true);
```

Comment:
This test would fail if you checked 

`TrustedServers.contains('http://cesiumjs.org:80/index.html')` 

right?  Perhaps we should just always require host and port, in all cases.